.github/ROADMAP.md

```diff
@@ -12,11 +12,10 @@ The team will release one minor version upgrade each quarter, or whenever a new
 
 
 2022 Q4:
-- 2D Semantic Segmentation task, the team focuses on DeepLabV3 and opens up contribution
-  for UNet/FCN.
-- 3D Object Detection task, the team focuses on providing point cloud data pipeline.
+- 2D Semantic Segmentation task, the team focuses on DeepLabV3, DeepLabV3+ and opens up contribution for UNet/FCN.
+- 3D Object Detection task, the team focuses on providing point cloud data augmentation pipeline, losses and metrics.
```

Comment:
Any scope to add 3D semantic segmentation? It might give opportunity to explore medical imaging data with kcv.

Reply:
it should, though we probably cannot take this until after 2023 Q1.